tests/CMakeLists.txt

```diff
@@ -122,6 +122,11 @@ FOREACH(_test ${_tests})
     MESSAGE("Ignoring invalid .prm '${_test_full}'...")
     CONTINUE()
   ENDIF()
+  IF (${_test_full} MATCHES "[.]x[.]prm$")
```

Comment:
Is this how you escape an actual dot in cmake? I guess it works as a character class, but think it would be easier to read if it was `\.` as in most other regex languages.